internals/parser.rst

```diff
@@ -833,9 +833,9 @@ Testing
 
 There are three files that contain tests for the grammar and the parser:
 
-* `Lib/test/test_grammar.py`.
-* `Lib/test/test_syntax.py`.
-* `Lib/test/test_exceptions.py`.
+* ``Lib/test/test_grammar.py``.
+* ``Lib/test/test_syntax.py``.
+* ``Lib/test/test_exceptions.py``.
```

Comment:
These were actual errors.

Reply:
Maybe just link the actual files (on GitHub instead)?

Reply:
That would be a good improvement for another PR.

Reply:
* #960